### ISSUE TITLE
Backport #721 into `2.2.x` branch

### ISSUE THREAD

**Issue by elefeint** [spring-cloud/spring-cloud-commons#725]
Issue surfaced in `spring-cloud-gcp` build -- using `spring-cloud-bus` at version `2.2.2.BUILD-SNAPSHOT` and spring-cloud-commons at `2.2.3.BUILD-SNAPSHOT` currently fails.

The use of the new utility [got backported](https://github.com/spring-cloud/spring-cloud-bus/pull/224) in `spring-cloud-bus`, but it does not yet exist in commons, leading to the following startup error:

```
java.lang.NoSuchMethodError: 'java.lang.String org.springframework.cloud.commons.util.IdUtils.getUnresolvedServiceId()'
	at org.springframework.cloud.bus.BusEnvironmentPostProcessor.postProcessEnvironment(BusEnvironmentPostProcessor.java:48)
```

#721 getting backported in `spring-cloud-commons` into 2.2.x branch would fix the issue.

**Comment by ryanjbaxter** [comment]
@elefeint #721 was merged into the 2.2.x branch and then merged forward into master

https://github.com/spring-cloud/spring-cloud-commons/blob/2.2.x/spring-cloud-commons/src/main/java/org/springframework/cloud/commons/util/IdUtils.java

**Comment by elefeint** [comment]
I see what happened -- we are still on `spring-cloud-commons:2.2.1.BUILD-SNAPSHOT`. I'll update the dependency.

Sorry for the noise!
